docs/chapters/ci-and-cd/pelux-ci-cd-jobs.rst

```diff
@@ -0,0 +1,81 @@
+PELUX Continuous Integration jobs
+=================================
+
+For the PELUX Baseline, the Jenkins instance at `pelux.io/jenkins
+<http://pelux.io/jenkins>`_ has some predefined jobs that we run regularly or on
+commit.
+
+Jenkins jobs
+------------
+The jobs available in our Jenkins set up can be divided into three categories,
+yocto builds, Software Factory builds and meta jobs.
+
+Many of the jobs we have are `multi-branch pipelines`_. In Jenkins terms, this
+means that it scans the repository it is configured for, and builds any branches
+and any pull requests. This makes it easy to build for sanity checking a pull
+request before pulling it in, while still automatically building the target
+branches whenever they are changed.
+
+Building pull requests works for at least both GitHub and GitLab, and PELUX uses
+GitHub for this.
+
+A multi-branch pipeline is triggered whenever there is a change to any of the
+branches or pull requests in the repository. Depending on network set up, this
+can be done by polling the repo or by using a webhook. Since the PELUX Jenkins
+is publicly available, we're using webhooks. This also ensures that there is no
+delay from push to build.
+
+Yocto builds
+^^^^^^^^^^^^
+The yocto builds currently do not archive any artifacts. This is for legal
+reasons, as archiving the artifacts on a public Jenkins instance would
+constitute "distribution" as described in the GNU GPL. When we build a binary
+release of PELUX, we will archive the artifacts and the complete and
+corresponding source code.
+
+We use a cache for all Yocto jobs except the nightly. The reason behind that is
```

Comment:
It might not be obvious to all what this is about. The clean build is explained, but it's not stated that the cache is a measure to enable shorter feedback loops. There's some potential to rewrite this paragraph to be more clear on the strategy of incremental for feedback, and clean for verification.

Reply:
Why to use a cache etc is mentioned in the Blueprint article that is in the same toctree.